x/ccv/provider/types/msg.go

```diff
@@ -887,3 +575,11 @@ func ValidateByteSlice(hash []byte, maxLength int) error {
 	}
 	return nil
 }
+
+func validateDeprecatedChainId(chainId string) error {
+	if strings.TrimSpace(chainId) != "" {
```

Comment:
What does this `TrimSpace`-check add? It seems we can just do `if chainId != "" { return fmt.Errorf("...") } return nil`.
This method can be removed, no reason why we would need to validate anything deprecated.

Reply:
good point

Reply:
Actually, if somebody leaves by mistake a space in the json file, the message should not be invalid. It's clearly that is not trying to use `chainId=" "` as an ID.